### PR TITLE
Formatted output and removed UT output

### PR DESCRIPTION
**Pull request by zhanggbj**
1. Formatted output with cmd.ui
2. Removed UT output(issue #45)

[#119336713](https://www.pivotaltracker.com/story/show/119336713)

### CODE REVIEW

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,
 I submitted code for this chore firstly as it may cause error "inconsistent definition" of type tablewriter in cmds/bmp and in common. Thanks for your review.

File: bin/test_unit [hunk 1]
```diff
@@ -14,6 +14,8 @@
   trap printStatus EXIT
   export GOPATH=$(godep path):$GOPATH
 
+  export RUN_BMPUT=TRUE
```
Comment (maximilien) [inline bin/test_unit]: I don't like the name, what about `BMP_UT_OUTPUT` which is more descriptive?

**Comment by maximilien** [comment]
Looks good to me, change the name, however. What's the error you see? Can you paste here?

**Comment by zhanggbj** [comment]
This is ready for review:)

The errors occurred when I tried to import "github.com/cloudfoundry-community/bosh-softlayer-tools/common/fakes" into cmds/bmp. It said "inconsistent definition for type tablewriter during import"

**Comment by maximilien** [comment]
Cool. Merging 👍

**Pull request merged**